examples/aerospike-cluster-values.yaml

```diff
@@ -0,0 +1,110 @@
+## Aerospike cluster size
+replicas: 3
+
+## Aerospike Database Docker image
+image:
+  repository: aerospike/aerospike-server-enterprise
+  tag: 7.1.0.0
+
+## Aerospike access control configuration
+aerospikeAccessControl:
+  users:
+    - name: admin
+      secretName: auth-secret
+      roles:
+        - sys-admin
+        - user-admin
+
+## Aerospike Configuration
+aerospikeConfig:
+  service:
+    feature-key-file: /etc/aerospike/secret/features.conf
+  security: {}
+
+  network:
+    service:
+      port: 3000
+    heartbeat:
+      port: 3002
+    fabric:
+      port: 3001
+
+  namespaces:
+    - name: test
+      replication-factor: 2
+      storage-engine:
+        type: device
+        devices:
+          - /test/dev/xvdf
+    - name: testMem
+      replication-factor: 1
+      storage-engine:
+        type: memory
+        data-size: 1073741824
+
+## Network policy
+aerospikeNetworkPolicy: {}
+
+## Pod spec
+podSpec:
+   multiPodPerHost: true
+
+## Rack configuration
+rackConfig:
+  namespaces:
+    - test
+  racks:
+    - id: 1
+      # Change to the zone for your k8s cluster.
+      zone: us-central1-c
+    - id: 2
+      # Change to the zone for your k8s cluster.
+      zone: us-central1-c
+
+## Storage configuration
+storage:
+  filesystemVolumePolicy:
+    cascadeDelete: true
+    initMethod: deleteFiles
+  blockVolumePolicy:
+    cascadeDelete: true
+  volumes:
+    - name: workdir
+      aerospike:
+        path: /opt/aerospike
+      source:
+        persistentVolume:
+          storageClass: ssd
```

Comment:
storage class procured using terraform is gp2/gp3 . ssd is not available

Reply:
fixed